### ISSUE TITLE
ca.crt and ca.key should be added in an independent secret

### ISSUE THREAD

**Issue by alinadonisa** [cockroachdb/cockroach-operator#491]
Move ca.crt in the same secret where ca.key is, so it can be mounted independently of the node.crt.

**Comment by keith-mcclellan** [comment]
No, we do not want to move ca.crt to the same secret as the ca.key, we need to create a cockroach.ca-crt secret that only includes the ca.crt.  No one other than a user minting a new certificate should ever be mounting ca.key

**Comment by alinadonisa** [comment]
@keith-mcclellan  this means we will have 4 secrets, right?

**Comment by keith-mcclellan** [comment]
yes.  We don't want to remove ca.crt from the node and client secrets, just have a 4th secret that only has the ca.crt.  Thanks
